pkgs/applications/radio/multimon-ng/default.nix

```diff
@@ -13,6 +13,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = lib.optionals stdenv.isLinux [ libpulseaudio libX11 ];
 
+  propagatedBuildInputs = [ sox ];
```

Comment:
Wouldn't it make more sense to wrap the binary and provide sox in the path?
This solution only works in `nix-shell`, but not when provided through the system environment or home manger.